### ISSUE TITLE
Card: Card details section as highlighted in red is not included in the tab chain (No visible focus provided). Hence, UI is not reachable via keyboard

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3811]
Description: Card details section as highlighted in red is not included in the tab chain (No visible focus provided). Hence, UI is not reachable via keyboard

Expected: Visible focus should be provided for all information related areas present within the UI in order to be identify by the screen reader. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/98772000-bbdd1300-240b-11eb-92c7-4b2e1d7a3df7.png)

![image](https://user-images.githubusercontent.com/32538291/98772016-bed80380-240b-11eb-9c9f-7ec2421bef82.png)

![image](https://user-images.githubusercontent.com/32538291/98772023-c39cb780-240b-11eb-9af8-4b0aa9537473.png)

![image](https://user-images.githubusercontent.com/32538291/98772030-c5ff1180-240b-11eb-87b5-4f62f8e94d4d.png)
